### PR TITLE
#38 read cooldowns from brainbot.ini

### PR DESCRIPTION
**Pull request by yohanesgultom**
Issues  #38 
* read cooldowns from brainbot.ini
* add brainbot.ini.example
* add brainbot.ini in .gitignore

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -10,15 +10,19 @@
...
 load_dotenv(
     dotenv_path=bot_dir / ".env"
 )  # Added path to support older versions of python-dotenv
 
-tell_me_to_cooldown = Cooldown(200)
-topic_cooldown = Cooldown(100)
+config = ConfigParser()
+config.read("brainbot.ini")
+
+tell_me_to_cooldown = Cooldown(config.getint("cooldowns", "tell_me_tos", fallback=200))
```
Comment (bsoyka) [inline main.py:24]:
```suggestion
tell_me_to_cooldown = Cooldown(config.getint("cooldowns", "tell_me_to", fallback=200))
```
Just this little typo and everything else looks good to me!

**Review by bsoyka — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (yohanesgultom) [reply main.py]: ah yeah, my bad. fixed it 😅

**Review by bsoyka — APPROVED** [review]
Fantastic! Thanks so much for taking the time to contribute! If you're participating in Hacktoberfest, this PR *will* count as this repo has the `hacktoberfest` topic.

I'll merge this in and get your name put up on the README for all to see. ;)

**Comment by bsoyka** [comment]
@all-contributors please add yohanesgultom for code

**Pull request merged**